CHANGELOG.md

```
@@ -1,4 +1,20 @@
 
+## 2.2.0 (22nd June 2026)
+
+### Features
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)